Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -103,8 +103,8 @@ protected override void OnInit(EventArgs e)
         private void HandleEditMode()
         {
             ///Forcing View Mode Until we can find a solution to prevent Edit Bar from appearing. 
-            ToggleUserMode("VIEW");
-            return;
+            //ToggleUserMode("VIEW");
+            //return;
```

Comment:
Please remove these commented lines. They're not needed.

Reply:
done